client/images/save.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="-4 -4 32 32" width="24">
```

Comment:
I don't think you need to add this icon, there's already an icon for this called `unsaved-changes-dot.svg`.

Reply:
This is the little diskette icon that the user presses to save the project. I couldn't find an icon for this. is there one available?

Reply:
Oh my bad! I got confused and thought this was the unsaved dot icon. All good then :)